source/qb64pe.bas

```diff
@@ -24637,22 +24700,23 @@ FUNCTION EvaluateNumbers$ (p, num() AS STRING)
             'Note, these are special cases and work with the number BEFORE the command and not after
```

Comment:
If you look at line `24628`, I think the addition of the non-prefixed entries to `OName()` creates problems because situations like that `Select Case OName(p)` now need to be checking for both.